common/transaction/types.go

```diff
@@ -21,14 +21,16 @@ import (
 	"github.com/ChainSafe/gossamer/core/types"
 )
 
+// Pool ValidTransaction
 type Pool map[common.Hash]*ValidTransaction
```

Comment:
Pool is a mapping of transaction hashes to ValidTransactions

Reply:
good description